### PR TITLE
feat: update to nx 16

### PR DESCRIPTION
**Pull request by Cammisuli**
closes: #62

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
           cache: 'npm'
 
       - name: Install npm dependencies
-        run: npm ci
+        run: npm ci --force
```
Comment (Cammisuli) [inline .github/workflows/tests.yml:35]: I'm putting this in temporary to get around the npm issue not liking betas in deps

**Comment by Cammisuli** [comment]
@TheUnderScorer this is ready for your review. It should be backwards compatible with nx 15 workspaces still, but give it a quick test 🙂

**Comment by orbziiharry** [comment]
Any chance of this being merged and released soon? Just migrated to nx 16 and would like to continue using semantic release

**Review by TheUnderScorer — APPROVED** [review]
LGTM, thanks @Cammisuli!

**Pull request merged**